charts/nexus-repository-manager/templates/test/test-check-logs.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.persistence.enabled }}
```

Comment:
This test confirms the chart is running the expected version of NXRM

Reply:
^^ maybe a nice comment for the head of the file?

Reply:
Done, I added a small comment for each test